src/modules/trade/hooks/useSwitchTokensPlaces.ts

```diff
@@ -0,0 +1,43 @@
+import { useCallback } from 'react'
+import { FractionUtils } from 'utils/fractionUtils'
+import { ExtendedTradeRawState } from '../types/TradeRawState'
+import { useIsWrapOrUnwrap } from './useIsWrapOrUnwrap'
+import { useTradeNavigate } from './useTradeNavigate'
+import { useWalletInfo } from 'modules/wallet'
+import { useTradeState } from './useTradeState'
+import { useDerivedTradeState } from './useDerivedTradeState'
```

Comment:
Why are the imports not sorted?

Reply:
It's because I forked from @nenadV91 branch and apparently it doesn't contain new linter rules

Reply:
Oooooh! Makes sense.
Not a problem, it'll be fixed (or force you to fix) when it's merged